doc/api/modules.md

```diff
@@ -529,34 +529,47 @@ const x = require('./x');
 console.log(x.a);
 ```
 
-#### exports alias
+#### exports shortcut
 <!-- YAML
 added: v0.1.16
 -->
 
-The `exports` variable that is available within a module starts as a reference
-to `module.exports`. As with any variable, if you assign a new value to it, it
-is no longer bound to the previous value.
+The `exports` variable that is available within a module starts as if
+`module.exports` had been assigned to it before the module was evaluated.
+
+It allows a shortcut, so that `module.exports.f = ...` can be written more
+succinctly as `exports.f = ...`. However, be aware that like any variable, if
+a new value is assigned to `exports`, it is no longer bound to `module.exports`:
+
+   module.exports.hello = true; // Exported from require of module
```

Comment:
Done, thanks, sorry for the churn, I'll be up to speed on current conventions soon.